source/custom/nixnet_converters.h

```diff
@@ -8,6 +8,8 @@ namespace pb_ = ::google::protobuf;
 
 namespace nixnet_grpc {
 
+#define EnetFrameHeaderLength sizeof(nxFrameEnet_t) - 1  // last byte in nxFrameEnet_t is u8 FrameData[1]
```

Comment:
nit: prefer `constexpr` and `UPPER_SNAKE_CASE` for constants C++.

```cpp
constexpr auto ENET_FRAME_HEADER_LENGTH = sizeof(nxFrameEnet_t) -1; // last byte in nxFrameEnet_t is u8 FrameData[1]
```

you could also include the static cast here if you always need a `u16`:

```cpp
constexpr auto ENET_FRAME_HEADER_LENGTH = static_cast<u16>(sizeof(nxFrameEnet_t) -1); // last byte in nxFrameEnet_t is u8 FrameData[1]
```

(I think the current code is a compile warning).

Reply:
Updated to use `constexpr` with `static_cast`